variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "resource_name_prefix" {
+  type        = string
+  description = "Prefix to give to names of infra created by this module, where applicable."
```

Comment:
This is used for container name AND the application names / credentials. 

But the convention is different for both; we can put anything we want as part of an application name/display name / description, but for a container there are some limitations -> https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftstorage :

> Lowercase letters, numbers, and hyphens.
>Start with lowercase letter or number. Can't use consecutive hyphens.

So i'm wondering if we want to allow app names and descriptions; I think they can be hardcoded (just worklytics-export-...) and then use the variable only for the container name, remanding in the description the limits the name should have.

Reply:
@davidfq I'd split this in two variables, even by default they could have same value.